### DISCUSSION TITLE
Add `trashed` event for SoftDeleted models

### DISCUSSION THREAD

**Discussion by calebdw** [owen-it/laravel-auditing#979] [Ideas]
Hello!

Thanks for such a great package! I'm in the middle of converting our custom auditing solution to use this package.

It seems that trashed models are shown as `deleted` in the audits, but this is confusing as the actual `forceDelete` action is also shown as `deleted`. Essentially, there's no way by looking at the table if a model uses soft deletes and if the record was actually deleted or just trashed

I think it would be great to instead have soft deleted models show the `trashed` state instead.

For example, the following code would produce these events in the audits:

```php
$m = ModelWithSoftDeletes::factory()->create(); // created
$m->delete(); // trashed
$m->restore(); // restored
$m->forceDelete(); // deleted
```

Thanks!

**Comment by angeljqv** [comment]
Feel free to make a PR, some member will review it and hopefully merge it.
but it sounds like a breaking change, because the delete event is arready being used on SoftDeleted models,
if someone waits for the delete event and suddenly gets trashed event?

**Reply by calebdw** [reply]
Both the `deleted` and `trashed` events are fired for soft deleted models, same for `deleted` and `forceDeleted`
